### DISCUSSION TITLE
Not all DigitalOcean spaces supported

### DISCUSSION THREAD

**Discussion by darkpixel** [requarks/wiki#5212] [Error / Bug Report] [answered]
It looks like Digital Ocean supports spaces in the SFO3 region, but the spaces storage plugin is a pick-list and not a free-form field.

Can someone add the SFO3 region to the dropdown?

**Comment by NGPixel** [comment, marked as answer]
Added in 485aa765d0b33d3165c6db5c7c3afeb2659f2b3f

It will be part of the next release.

Note that you can use the "S3 Generic" storage module in the meantime to manually enter the URL to sfo3

**Reply by darkpixel** [reply]
No worries--I took the "even more lazier" route and just created a new space in SFO2 for the time being. :)
